LICENSE

```diff
@@ -1,160 +1,674 @@
-
```

Comment:
This feels like it should be in its own/separate patchset? Did this file get added by accident?

Reply:
We noticed this old license when searching for "Dashboard", so took the opportunity the correct this - agree it probably should have been a separate task